### ISSUE TITLE
[NERSC] Create a new python env for webapps

### ISSUE THREAD

**Issue by heather999** [LSSTDESC/desc-help#45]
**Description**
We are considering creating a new DESC python environment dedicated to webapps.  We can use this issue to collect a list of what we want to see included.
We could also set this up as a docker image and install that at NERSC and other places - rather than continuing to create local installs.

Choose all applicable topics by placing an 'X' between the [ ]:
- [X] python

**Comment by yymao** [comment]
For DESCQA web env we only need `jinja2` and `pyyaml`.

**Comment by johannct** [comment]
django?

**Comment by yymao** [comment]
DESCQA web doesn't use `django` but it's a commonly-used package for web backend, so good to have in a web env.  To that end, `flask` would be good to have.

**Comment by heather999** [comment]
There's a new GH repo [desc-python-webapps](https://github.com/LSSTDESC/desc-python-webapps) that includes both an installation script and can create a docker image containing a miniconda and the above mentioned packages.  This is set up using the base environment, no other conda environment is created.  I've installed this at NERSC: `/global/common/software/lsst/common/miniconda/desc-python-webapps`  It can be enabled by updating PATH to point to `/global/common/software/lsst/common/miniconda/desc-python-webapps/bin`

I also built a docker image and installed it into shifter at NERSC as `lsstdesc/desc-python-webapps`
Updating descqa's webapp to use shifter would probably involve moving to SPIN.  Maybe a sprint week project :)  This would be worthwhile, as this would make the move to Perlmutter irrelevant.
